packages/components/src/components/accordion/_accordion.scss

```diff
@@ -199,7 +200,8 @@
 
     .#{$prefix}--accordion__arrow {
       transform: rotate(-90deg) #{'/*rtl:ignore*/'};
-      fill: $ui-05;
+      fill: $border-inverse;
```

Comment:
Should be `$icon-primary`

Reply:
This should have been using `$icon-01` not `$ui-05` its the same value but different use cases. So the migration isn't straight forward.